recipes/_omnibus_toolchain.rb

```diff
@@ -17,10 +17,12 @@
 # limitations under the License.
 #
 
+arch = (windows_arch_i386? || i386?) ? 'i386' : node['architecture']
```

Comment:
node['architecture'] is not in ohai, so node['kernel']['machine'] can be used instead